.github/workflows/package.yml

```diff
@@ -22,7 +22,7 @@ jobs:
       - name: Checkout
         uses: actions/checkout@v4
         with:
-          token: ${{ secrets.PAT }}
+          token: ${{ secrets.RHACS_BOT_GITHUB_TOKEN }}
```

Comment:
Why do we need a token here but not in `.github/workflows/update-main-version.yml`? Shouldn't we need it there as well for the `git push`?

Reply:
Ah, we do need it, good catch